languages/CRB.groovy

```diff
@@ -0,0 +1,90 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import com.ibm.dbb.metadata.*
+import com.ibm.dbb.dependency.*
+import com.ibm.dbb.build.*
+import com.ibm.dbb.build.report.records.*
+import com.ibm.dbb.build.report.*
+import groovy.transform.*
+
+/*
+The language script is for running the CICS tool (zrb) on CICS definitions in YML (yaml)
+format.
+The crb.properties file should provide 
+ - the zrb location on the host
+ - the path to the Model file 
+ - the application constraints file
+The script will get the resource CICS yml file names from the buildlist and proceed to execute 
+the zrb executable on them. This will create a CSD formatted file, which can be stored as an artifact 
+or can be run on Z/OS through DFHCSDUP or CicsDef.groovy script
+*/
+
+@Field BuildProperties props = BuildProperties.getInstance()
+@Field def buildUtils= loadScript(new File("${props.zAppBuildDir}/utilities/BuildUtilities.groovy"))
+
+// verify required build properties
+buildUtils.assertBuildProperties(props.crb_requiredBuildProperties)
+
+List<String> buildList = argMap.buildList
+
+println("** Processing files mapped to ${this.class.getName()}.groovy script")
```

Comment:
With the next release of zAppBuild, the language script have a progress indicator. You can find it already in the  `develop` branch contributed with https://github.com/IBM/dbb-zappbuild/commit/04e6118030f1764bd82f637223f6fdcbfac52ce9. Can we please have the same logic here?

Reply:
Changed to show progress indicator